cameraview/src/main/java/com/otaliastudios/cameraview/engine/Camera2Engine.java

```diff
@@ -1353,6 +1355,11 @@ public void setPlaySounds(boolean playSounds) {
         mPlaySoundsTask = Tasks.forResult(null);
     }
 
+    @Override
+    public void setPreviewFrameRateExact(boolean previewFrameRateExact) {
+        mPreviewFrameRateExact = previewFrameRateExact;
+    }
+
```

Comment:
See my comment above about simply doing this in CameraBaseEngine and having engines call `getPreviewFrameRateExact()`. This getter will also help in the CameraView getter which I also mentioned before.